include/RAJA/RAJA.hxx

```diff
@@ -152,6 +152,10 @@
 
 #endif  // defined(RAJA_ENABLE_NESTED)
 
+#if defined(RAJA_ENABLE_SCAN)
+
```

Comment:
The guard should only be needed for the inclusion of `exec-cuda/raja_cuda.hxx` (`#include "RAJA/exec-cuda/scan_cuda.hxx"`)